ml-data-hub-plugin/src/test/groovy/com/marklogic/gradle/task/UpdateIndexesTaskTest.groovy

```diff
@@ -53,43 +54,44 @@ class UpdateIndexesTaskTest extends BaseTest {
 		// Copying Order.entity.json file to plugins/entities/my-unique-order-entity directory
 		String entityStream = new File("src/test/resources/update-indexes/my-unique-order-entity.entity.json").getAbsolutePath()
 		Files.copy(new File(entityStream).toPath(), entityDir.toPath().resolve("my-unique-order-entity.entity.json"), StandardCopyOption.REPLACE_EXISTING)
-		
+
 		// Loading modules to databases
 		runTask('mlLoadModules')
-		
+
 		// Copying Staging and Final database Index info files to src/main/entity-config dir
 		Path dir = hubConfig().getEntityDatabaseDir()
 		if (!dir.toFile().exists()) {
 			dir.toFile().mkdirs()
 		}
-		
+
 		File dstFile = Paths.get(dir.toString(), HubConfig.STAGING_ENTITY_DATABASE_FILE).toFile()
 		String entityConfigStream = new File("src/test/resources/update-indexes/staging-database.json").getAbsolutePath()
 		Files.copy(new File(entityConfigStream).toPath(), dstFile.toPath(), StandardCopyOption.REPLACE_EXISTING)
-		
+
 		dstFile = Paths.get(dir.toString(), HubConfig.FINAL_ENTITY_DATABASE_FILE).toFile();
 		entityConfigStream = new File("src/test/resources/update-indexes/final-database.json").getAbsolutePath()
 		Files.copy(new File(entityConfigStream).toPath(), dstFile.toPath(), StandardCopyOption.REPLACE_EXISTING)
-		
+
 		dir = hubConfig().getHubConfigDir()
 		dstFile = Paths.get(dir.toString(), "databases", "job-database.json").toFile()
 		entityConfigStream = new File("src/test/resources/update-indexes/job-database.json").getAbsolutePath();
 		Files.copy(new File(entityConfigStream).toPath(), dstFile.toPath(), StandardCopyOption.REPLACE_EXISTING);
 	}
-	
+
+    @IgnoreIf({ System.getProperty('mlIsProvisionedEnvironment') })
 	def "test to deploy indexes to STAGING/FINAL/JOBS Database"() {
```

Comment:
You can use @IgnoreIf({hubConfig().isProvisionedEnvironment}) to ignore this test

Reply:
Yeah, please do, incase we need to override the gradle sys props in the future

Reply:
I tried them all but in vain.
@IgnoreIf({hubConfig().isProvisionedEnvironment})
@IgnoreIf({hubConfig(). getIsProvisionedEnvironment()}) //this is what I had before when the test failed in Jenkins run
@IgnoreIf({ isProvisioned() }) where isProvisioned is a boolean method evaluating hubConfig().isProvisionedEnvironment == true
 I get 

`com.marklogic.gradle.task.UpdateIndexesTaskTest > initializationError FAILED
    org.spockframework.runtime.extension.ExtensionException: Failed to evaluate @IgnoreIf condition
        Caused by:
        groovy.lang.MissingMethodException: No signature of method: com.marklogic.gradle.task.UpdateIndexesTaskTest$__spock_feature_1_0_closure1.hubConfig() is applicable for argument types: () values: []
`

Reply:
Spock 1.1 is based on JUnit4, so org.junit.jupiter.api.Assumptions will not work... org.junit.Assume will work..
I am ok with using @IgnoreIf({ System.getProperty('mlIsProvisionedEnvironment') }) as well.